fem/transfer.hpp

```diff
@@ -307,62 +289,62 @@ class L2ProjectionH1GridTransfer : public GridTransfer
       /// element space.  Refined mesh should be a uniform refinement of the coarse
       /// mesh. Coefficients are computed from the mass conservative left-inverse
       /// prolongation operation.  This functionality is also provided as an
-      /// Operator by L2ProlongationH1. Builds the prolongation operator if P is
-      /// NULL.
-      void Prolongate(const Vector& x, Vector& y) const;
+      /// Operator by L2Prolongation.
+      virtual void Prolongate(const Vector& x, Vector& y) const;
       /// Maps <tt>x</tt>, dual field coefficients defined on a coarse mesh with
       /// a higher order H1 finite element space, to <tt>y</tt>, dual field
       /// coefficients defined on a refined mesh with a low order H1 finite
       /// element space.  Refined mesh should be a uniform refinement of the coarse
       /// mesh. Coefficients are computed from the transpose of the mass
       /// conservative left-inverse prolongation operation.  This functionality is
-      /// also provided as an Operator by L2ProlongationH1. Builds the prolongation
-      /// operator if P is NULL.
-      void ProlongateTranspose(const Vector& x, Vector& y) const;
+      /// also provided as an Operator by L2Prolongation.
+      virtual void ProlongateTranspose(const Vector& x, Vector& y) const;
+      virtual void SetTolerance(double p_rtol_, double p_atol_);
```

Comment:
It might be better to split this into `SetRelTol` and `SetAbsTol` just like in MFEM's iterative solvers.

Reply:
Good idea.  I put this in the latest commit.